charts/rucio-server/templates/deployment.yaml

```diff
@@ -86,6 +91,34 @@ spec:
         secret:
           secretName: {{ .Release.Name }}-server-cafile
 {{- end }}
+{{- end }}
+{{- if .Values.policyPackages.enabled }}
+      initContainers:
+        - name: install-policy-packages
+          # A slim Python image could be used instead, but using the same image as the base image for the main container ensures that the same Python version is used
+          image: almalinux:9
+          command:
+            - "sh"
+            - "-e"
+            - "-c"
+            - |
+              export PYTHONPATH={{ .Values.policyPackages.mountPath }}:$PYTHONPATH
+              {{- range .Values.policyPackages.packages }}
+              if python3 -c "from importlib.metadata import version; assert version({{ .moduleName | squote }}) == {{ .version | squote }}"
+              then
+                  echo "module {{ .moduleName }} version {{ .version }} is already installed"
+              else
+                  if [[ {{ .requirement | squote }} == git+* ]]
+                  then
+                      dnf install --assumeyes git-all
+                  fi
```

Comment:
This is entirely a matter of personal style, but I prefer the following structure, which condenses the code vertically:

```shell
if condition; then
    block
fi
```